t/unit-tests/unit-test.c

```diff
@@ -2,6 +2,9 @@
 #include "parse-options.h"
 #include "string-list.h"
 #include "strvec.h"
+#include "strbuf.h"
+#include "hex.h"
+#include "unit-test.h"
```

Comment:
Let's keep headers sorted alphabetically (except for "unit-test.h", which is expected to come first. You also got that one duplicated now.